specs/phase0/beacon-chain.md

```diff
@@ -1036,7 +1036,7 @@ def initiate_validator_exit(state: BeaconState, index: ValidatorIndex) -> None:
 
     # Compute exit queue epoch
     exit_epochs = [v.exit_epoch for v in state.validators if v.exit_epoch != FAR_FUTURE_EPOCH]
-    exit_queue_epoch = max(exit_epochs, default=compute_activation_exit_epoch(get_current_epoch(state)))
+    exit_queue_epoch = max(exit_epochs + [compute_activation_exit_epoch(get_current_epoch(state))])
```

Comment:
Thinking if it is worth doing max(max()) to emphasize the intention.
```suggestion
    exit_queue_epoch = max(max(exit_epochs), compute_activation_exit_epoch(get_current_epoch(state)))
```

Reply:
> Thinking if it is worth doing max(max()) to emphasize the intention.

Please do.  The `+` notation is unobvious for anyone not versed in python.

Reply:
Agreed that `max(max(), .)` is slightly more readable.

Reply:
`max` of empty list fails and this happens early in the consensus when doing `max(exit_epochs)`

Reply:
What do you think using default in max here?
```suggestion
    exit_queue_epoch = max(
        max(exit_epochs, default=0),  # default 0 if exit_epochs is empty
        compute_activation_exit_epoch(get_current_epoch(state)), # Make exit_queue_epoch not less than this number
    )
```

Reply:
In hindsight `max(exit_epochs + [compute_activation_exit_epoch(get_current_epoch(state))])` is probably readable enough. I got confused and/or tried to be too clever with the `default=` argument.

Reply:
going with the array concat!
Thanks for the review everyone :)